src/client/app/containers/editor.mode.ts

```diff
@@ -115,7 +131,7 @@ export class EditorMode {
         }
 
         if (isOfficeHost(this.snippet.host)) {
-            if (!isInsideOfficeApp()) {
+            if (!isInsideOfficeApp() && !this.isEditorTryIt) {
```

Comment:
This `and` logic of two negatives is confusing.  What is the intention?  At the very least, could you write it out as `let blah = ...` `if (blah) {...}`